.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
           - 6379:6379
     strategy:
       matrix:
-        ruby-version: [2.7, 2.6]
+        ruby-version: ['3.0', 2.7, 2.6]
```

Comment:
Why is 3.0 in quotes but the others are not? If it's because the zero gets truncated, or something please quote the other values for consistency.

Reply:
Yes that's exactly the [bug](https://github.com/actions/runner/issues/849). Fix pushed.